ci/scripts/repeat-new-tests.sh

```diff
@@ -70,19 +70,19 @@ CHANGED_FILES_ARRAY=( $UNIT_TEST_CHANGES $INTEGRATION_TEST_CHANGES $DISTRIBUTED_
 NUM_CHANGED_FILES=${#CHANGED_FILES_ARRAY[@]}
 
 echo "${NUM_CHANGED_FILES} changed test files"
-for T in ${CHANGED_FILES_ARRAY[@]}; do
+for T in "${CHANGED_FILES_ARRAY[@]}"; do
   echo "  ${T}"
 done
 
-if [[  "${NUM_CHANGED_FILES}" -eq 0 ]]
+if [[  "${NUM_CHANGED_FILES}" -eq "0" ]]
```

Comment:
Shouldn't be necessary since `-eq` acts on arithmetic expressions. Either way it works though.

Reply:
Right.